iron-overlay-manager.html

```diff
@@ -220,6 +220,7 @@
       }
       this.backdropElement.style.zIndex = this._getZ(overlay) - 1;
       this.backdropElement.opened = !!overlay;
+      this.backdropElement.prepare();
```

Comment:
Property observers are not triggered until the element is attached https://github.com/Polymer/polymer/issues/4526
I should probably add a comment here..!